sites/hcinnovationgroup.com/server/templates/content/index.marko

```diff
@@ -4,6 +4,9 @@ import contentKeyValues from "@endeavor-business-media/package-common/gam/conten
 import queryFragment from "../../graphql/fragments/content-list";
 import GAM from "../../../config/gam";
 
+$ const formatCurrentcy = function(number) {
+  return '$' + new Intl.NumberFormat().format(number)
```

Comment:
This is used twice. Move to a common utility function an import. Use template literal and an arrow function:

```js
module.exports = (number) => {
  if (number == null) return '';
  return `$${new Intl.NumberFormat().format(number)}`;
}
```